tests/pytorch/mock_sparse2/test_reduction.py

```diff
@@ -0,0 +1,131 @@
+import operator
+import sys
+
+import backend as F
+
+import dgl.mock_sparse2 as dglsp
+import pytest
+import torch
+
```

Comment:
Do we need to skip the test on windows as in other test files?

Reply:
I'll try if it works on Windows first and then decide (now that I upgraded Windows CI to PyTorch 1.12)

Reply:
Yeah it seems that DGL sparse still has trouble compiling on Windows.  I think it's a CMake problem.  I'll fix it in later PRs.